doc/tutorials/tutorial_X8.rst

```diff
@@ -1,18 +1,17 @@
 .. _starship:
 
-Executing programs on photonic hardware
-=======================================
+Executing programs on the X8 device
+===================================
 
 In this tutorial, we demonstrate how Strawberry Fields can execute
-quantum programs on remote photonic hardware, via the Xanadu cloud platform. Topics
+quantum programs on the remote photonic X8 device, via the Xanadu cloud platform. Topics
 covered include:
 
 * Configuring Strawberry Fields to access the cloud platform,
-* Using the :class:`~.RemoteEngine` to execute jobs remotely (e.g., on a quantum chip),
-* Managing submitted jobs,
+* Using the :class:`~.RemoteEngine` to execute jobs remotely on chip ``X8_01``,
```

Comment:
One thing that came to my mind when I saw this:
"So is the chip `X8` or `X8_01`?"

Reply:
> explain the nomenclature of chips, i.e. X to encode architecture, 8 for number of spatial times frequency modes, _01 as a running number to identify different physical chips that are used in the hardware implementation, and finish with a comment that the specific chip instance must be chosen for running jobs on it.

I think this might be the best approach.